examples/jsm/nodes/core/VarNode.js

```diff
@@ -80,10 +80,8 @@ class VarNode extends Node {
 
 export default VarNode;
 
-export const label = nodeProxy( VarNode );
-export const temp = label;
+export const temp = nodeProxy( VarNode );
```

Comment:
If we remove `label` from VarNode then I sugget to also remove  `name` argument from it?